### PR TITLE
Copy: Spanish copy for Login.gov and CC

### LINKED ISSUE #611: Spanish: Translate new copy and upload translations for Courtesy Cards
Current status `09/26/22`

Copy document: https://docs.google.com/document/d/11s3oFXwpRykFoRSpOeQ3MBLLIq2CXQjI/edit

----

What else is missing from that doc that needs to be translated:

image

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #611

## What this PR does
- Add Spanish copy for the following pages:

1. Agency Index
2. Eligibility
3. Eligibility Start: Login.gov and CC
4. Eligibility Confirm: CC
5. Enrollment: Login.gov and CC
6. Enrollment Success: Login.gov and CC

- FYI: Help page will come in a different PR

## New terms/styles

- Beneficios de Cal-ITP - Cal-ITP Benefits (name of the app/website/program)
- Tarjeta de Cortesía de MST - MST Courtesy Card
- Tarjeta de Cortesía - Courtesy Card
- número de tarjeta de cortesía de MST - MST Courtesy Card number
- número de tarjeta de cortesía - Courtesy Card number
- titular de tarjeta de cortesía de MST actual - MST Courtesy Cardholder

## What is missing....

- Courtesy Card: Unverified page

```
msgid "core.pages.user_error.title"
msgstr "TODO: Invalid state"

msgid "core.pages.user_error.headline"
msgstr "TODO: You may have reached this page on accident."

msgid "core.pages.user_error.p[0]"
msgstr ""
"TODO: To get started with Cal-ITP Benefits please click the button below, "
"and you will be directed to the beginning of the enrollment process. "

msgctxt "image alt text"
msgid "core.icons.calendarcheck"
msgstr "TODO: A calendar page icon with a check mark"
```

### CODE REVIEW

**Comment by machikoyasuda** [comment]
@thekaveman Ready for review

See PR note above for caveat

**Review by thekaveman — COMMENTED** [review]
Here's a scenario where Title Case on the buttons doesn't necessarily make sense:

image

vs.

image

File: benefits/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -554,16 +558,16 @@ msgid "core.buttons.retry"
...
-msgstr "¡Éxito! Su descuento está ahora vinculado a su tarjeta bancaria."
+msgstr "¡Felicidades! Su descuento ahora está vinculado a su tarjeta bancaria."
 
 #, python-format
 msgid "enrollment.pages.success.helplink%(link)s"
 msgstr ""
-"TODO: Thank you for using Cal-ITP Benefits! For more information check out "
-"our <a href=\"%(link)s\" rel=\"noopener noreferrer\">Help Page</a>."
+"Gracias por usar los Beneficios de Cal-ITP. Para más información, consulte "
+"nuestro our <a href=\"%(link)s\" rel=\"noopener noreferrer\">Help Page</a>."
```
Comment (thekaveman) [inline benefits/locale/es/LC_MESSAGES/django.po:570]: Link text here should be `Página de Ayuda`

**Review by thekaveman — CHANGES_REQUESTED** [review]
One minor fix

**Comment by machikoyasuda** [comment]
Update: Unverified page is now translated 
image

File: benefits/locale/es/LC_MESSAGES/django.po (hunk 1 above)
Comment (machikoyasuda) [reply benefits/locale/es/LC_MESSAGES/django.po]: Done https://github.com/cal-itp/benefits/pull/1115/commits/dea38b1b7adb4497f6a81f4cfb6d8809d6a688d7

**Review by thekaveman — APPROVED** [review]
## Note on Login.gov

_Not sure we have any ability to control this_: if a user views Benefits in Spanish, then leaves for Login.gov, **unless their browser is configured with Spanish as the default language**, then when the user comes back to Benefits, it will have switched back to English.

If the user does have Spanish as the default language in their browser, then Benefits (and Login.gov) will display in Spanish by default.

## Note on form validation

Similarly, if the user has their browser set to display _its own UI in Spanish_, e.g. for Chrome:

image

Then the form validation messages will also display in Spanish:

image

image

image

**Pull request merged**